library/src/main/java/com/novoda/downloadmanager/lib/BatchRepository.java

```diff
@@ -38,22 +39,30 @@
 
     private final ContentResolver resolver;
     private final DownloadDeleter downloadDeleter;
+    private final Uri batchContentUri;
+    private final Uri allDownloadsContentUri;
 
-    public BatchRepository(ContentResolver resolver, DownloadDeleter downloadDeleter) {
+    public static BatchRepository newInstance(ContentResolver resolver, DownloadDeleter downloadDeleter) {
```

Comment:
yep it's purely for the constants which have only been made fields because of the android testing framework and Uris.

Seemed a little unfair to enforce the full constructor, the other option is two provide two constructors.....